returnn/hdf.py

```diff
@@ -309,13 +309,20 @@ def __init__(
         self.allophone_file = allophone_file
         self.state_tying_file = state_tying_file
         self.out_hdf_files = [self.output_path(f"data.hdf.{d}", cached=False) for d in range(len(alignment_caches))]
+        self.out_excluded_segments = self.output_path(f"excluded.segments", cached=False)
         self.returnn_root = returnn_root
         self.data_type = data_type
         self.rqmt = {"cpu": 1, "mem": 8, "time": 0.5}
 
     def tasks(self):
         yield Task("run", rqmt=self.rqmt, args=range(1, (len(self.alignment_caches) + 1)))
 
+    def _write_text(self, path, segment_list):
```

Comment:
`staticmethod`?

`i6_core.util.write_paths_to_file` has a very similar logic. maybe you can reuse that

Reply:
> `i6_core.util.write_paths_to_file` has a very similar logic. maybe you can reuse that

I cannot use this function, because the writing flag is hardcoded inthere to be `w`, which means that it will overwrite the file, meaning you will find the output of the last sub_task that wrote to the file.

Reply:
However, now with the `merge` task that should not be a problem anymore.